project_task_pull_request/tests/test_project_task_pull_request.py

```diff
@@ -57,3 +57,12 @@ def test_write_allowed_with_pr(self):
         self.task_3.write({'stage_id': 6, })
         self.task_3.refresh()
         self.assertEquals(6, self.task_3.stage_id.id)
+
+    def test_pr_uri(self):
```

Comment:
This test is pointless because it is covering standard "write"->"read" behaviour. Not the current module flow.
So you should remove it.